tests/test_cli_compile.py

```diff
@@ -2494,6 +2497,181 @@ def test_all_extras_fail_with_extra(fake_dists, runner, make_module, fname, cont
     assert exp in out.stderr
 
 
+def _parse_deps(text: str, keys: Iterable[str]) -> dict[str, str | None]:
+    """Parse `pip-compile` output and return mapping from package name to package version
+
+    :param text: output from `pip-compile`
+    :param keys: names of expected packages. If not in text the version will be set to `None`.
```

Comment:
It's RST
```suggestion
    """Parse ``pip-compile`` output and return mapping from package name to package version

    :param text: output from ``pip-compile``
    :param keys: names of expected packages. If not in text the version will be set to :data:`None`.
```

Reply:
The `:data:` prefix does not build when I try but other than that these are good suggestions.

Reply:
Ah, apparently, intersphinx isn't connected, which is why it can't find `None` with the `:data:` role — is documented in the CPython docs.